crates/bevy_ecs/src/schedule/executor/simple.rs

```diff
@@ -98,6 +98,8 @@ impl SystemExecutor for SimpleExecutor {
 }
 
 impl SimpleExecutor {
+    /// Creates a new simple executor for use in a [`Schedule`](super::super::Schedule).
```

Comment:
This should use an absolute link instead of multiple `super`s.

Try to prefer footnoting the links too so it's more readable in source form.